tests/Cms/App/AppTest.php

```diff
@@ -1369,22 +1372,21 @@ public function testControllerRepresentation()
 		]);
 
 		Page::factory([
-			'slug' => 'test',
+			'slug'     => 'test',
 			'template' => 'another'
 		]);
 
-		ob_start();
-		$app->controller('another.json', [], 'json');
```

Comment:
`Question:` Was it ever right to have this test calling with `another.json` as name and `json` as contentType instead of `another` and `json`?

Reply:
No, that looks wrong.